KarhooUISDK/UIComponents/LegalNotice/LegalNoticeStringBuilder.swift

```diff
@@ -13,26 +13,36 @@ struct LegalNoticeStringBuilder {
     
     func getLegalNotice() -> NSAttributedString {
         let paragraphStyle = NSMutableParagraphStyle()
-        paragraphStyle.alignment = .center
+        paragraphStyle.alignment = .left
         paragraphStyle.lineSpacing = UIConstants.Spacing.xSmall
         
-        let titleForLink: String = UITexts.Booking.legalNoticeTitle
+        let titleForLink: String = "quo voluptas nulla pariatur" // UITexts.Booking.legalNoticeTitle
         let link = UITexts.Booking.legalNoticeLink
         let legalNoticeText = NSMutableAttributedString()
         let regularAttributes: [NSAttributedString.Key: Any] = [
             .font: KarhooUI.fonts.captionRegular(),
             .foregroundColor: KarhooUI.colors.text,
             .paragraphStyle: paragraphStyle
         ]
-        let linkAttibutes: [NSAttributedString.Key: Any] = [
+        var linkAttibutes: [NSAttributedString.Key: Any] = [
             .font: KarhooUI.fonts.captionRegular(),
             .foregroundColor: KarhooUI.colors.accent,
             .underlineStyle: NSUnderlineStyle.single.rawValue,
             .underlineColor: KarhooUI.colors.accent,
             .paragraphStyle: paragraphStyle,
             NSAttributedString.Key(rawValue: "link"): "link"
         ]
-        let fullText = String(format: NSLocalizedString(UITexts.Booking.legalNoticeText, comment: ""), titleForLink)
+        if let linkUrl = URL(string: link) {
+            linkAttibutes[.link] = linkUrl
+        }
+        var text: String {
+            #if DEBUG
```

Comment:
let's have legal notice officially this on one of the test flows and remove this if/else